### ISSUE TITLE
Make operational characteristics inputs more user friendly

### ISSUE THREAD

**Issue by anamileva** [blue-marble/gridpath#783]

**Comment by milindsmart** [comment]
Pulling up an old post I made on slack for easier accessibility.

Project operational characteristics is perhaps the widest table in the database, except perhaps for scenarios. However, the `opchar` table supports some fairly inherent grouping which could be used to split it up. A narrower table would definitely help in easier modelling using the CSVs, but also could result in some more modularisation. Here's my view on the split.

First, we can have an overview table which can links to project level subscenarios - we could call it `project_operational_overview`

- project
- technology
- operational_type
- last_commitment_stage
- balancing_type_project
- fuel
- variable_generator_profile_scenario_id
- hydro_operational_chars_scenario_id
- heat_rate_curves_scenario_id
- variable_om_curves_scenario_id
- startup_chars_scenario_id

Next, the main project operational parameters and characteristics - this would continue to be called project_operational_chars, or maybe it can be `project_operational_props`

- project
- unit_size_mw
- min_stable_level_fraction
- aux_consumption_frac_capacity
- aux_consumption_frac_power
- charging_efficiency
- discharging_efficiency
- variable_om_cost_per_mwh
- startup_cost_per_mw
- shutdown_cost_per_mw
- startup_plus_ramp_up_rate
- shutdown_plus_ramp_down_rate
- ramp_up_when_on_rate
- ramp_down_when_on_rate
- ramp_up_violation_penalty
- ramp_down_violation_penalty
- min_up_time_hours
- min_up_time_violation_penalty
- min_down_time_hours
- min_down_time_violation_penalty
- curtailment_cost_per_pwh - perhaps also change other `mwh` columns to `pwh`
- charging_capacity_multiplier
- discharging_capacity_multiplier
- minimum_duration_hours - To be moved as per #771 
- maximum_duration_hours - To be moved as per #771 
- startup_fuel_mmbtu_per_mw - Can we instead have a `startup_fuel` column referring to an entry in the fuels table?

Last, the advanced operational characteristics need be provided only if reserves are modelled - this could be called `project_reserves_opchar`. This means the features `lf_reserves_up`, `lf_reserves_down`, `regulation_up`, and `regulation_down` are activated.

- project
- lf_reserves_up_derate
- lf_reserves_down_derate
- regulation_up_derate
- regulation_down_derate
- frequency_response_derate
- spinning_reserves_derate
- lf_reserves_up_ramp_rate
- lf_reserves_down_ramp_rate
- regulation_up_ramp_rate
- regulation_down_ramp_rate
- frequency_response_ramp_rate
- spinning_reserves_ramp_rate
